src/test/java/LianoHomeworkTests.java

```diff
@@ -0,0 +1,30 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class LianoHomeworkTests extends BaseTest {
+
+    @Test
+    public static void NavigationTest() {
+
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+        String url = "https://bbb.testpro.io/";
+        driver.get(url);
+
+        WebElement registrationLink = driver.findElement(By.id("hel"));
+        registrationLink.click();
+
+        WebElement registrationButton = driver.findElement(By.xpath("//input[@type='submit'][@value='Register']"));
```

Comment:
Consider simplifying code (locator):

For example `//input[@type='submit']`

![image](https://user-images.githubusercontent.com/104761482/209931689-fd89313d-e8b2-49f3-8330-241e1c368ba6.png)